src/promptflow-tools/README.dev.md

```diff
@@ -17,3 +17,43 @@ pip install pytest pytest-mock
 
 Use this [workflow](https://github.com/microsoft/promptflow/actions/workflows/tools_secret_upload.yml) to upload secrets in key vault. The secrets you uploaded would be used in [tools tests](https://github.com/microsoft/promptflow/actions/workflows/tools_tests.yml). Note that you only need to upload the SECRETS.
   > [!NOTE] After triggering the workflow, kindly request approval from Promptflow Support before proceeding further.
+
+## PR check-in criteria
+We provide a set of criteria that you can use to review your own code before submitting your pull request.
```

Comment:
Here's a friendly heads-up! We've got some criteria for you to self-review your code changes. It's a great way to double-check your work and make sure everything is in order before you share it. Happy coding! 😊👨‍💻👩‍💻